lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -759,19 +774,11 @@ def _post(self, params: ApiParams, _) -> Response:
         # non-terminal errors, but this requires some cleanup work on the
         # pyesbulk side.
         if error_count > 0:
-            self.logger.error(
-                "{}:dataset {}: {} successful document actions and {} failures: {}",
-                klasname,
-                dataset,
-                count - error_count,
-                error_count,
-                json.dumps(report),
-            )
+            auditing["status"] = AuditStatus.WARNING
+            auditing["reason"] = AuditReason.CONSISTENCY
+            attributes["message"] = f"Unable to {self.action} some indexed documents"
```

Comment:
If `error_count == count`, then we don't have a "consistency" problem, we have a total failure (or, possibly, a complete lack of success).  Unless we're handling that situation differently, I'm thinking that line 779 should be a little more specific than just "some".  (E.g., at the very least, it seems like we should use the same message at line 779 as we're gonna use at line 782; but, if the "OK count" is actually zero, perhaps we should be going down a different path altogether.)

Reply:
Failure means we had Elasticsearch bulk operation issues, SQL issues or (in the case of delete) file system errors preventing us from completing the operation. We're not reporting it as a 400, mostly because, whatever went wrong, it might be transient and the client can re-try.

But, yeah; in this case the distinction between `INTERNAL` and `CONSISTENCY` is squishy, and I suppose I could have gone either way. I dunno; maybe `INTERNAL` would be better. 🍤